modules/ca/src/client/udpiiu.cpp

```diff
@@ -725,6 +725,33 @@ bool udpiiu :: searchRespAction (
         serverAddr.ia.sin_addr = addr.ia.sin_addr;
     }
 
+#ifdef MIN_IOC_CA_VER_WHEN_REDIRECT
+    // Try to detect the nameserver redirecting us to a different IP
+    if (memcmp(&addr.ia.sin_addr, &serverAddr.ia.sin_addr, sizeof(struct in_addr)) != 0) {
```

Comment:
Does it make sense to avoid memcmp() here and use something similar
to the code in repeater.cpp ?

`inline bool repeaterClient::identicalAddress ( const osiSockAddr &fromIn )
{
    if ( fromIn.sa.sa_family == this->from.sa.sa_family ) {
        if ( fromIn.ia.sin_port == this->from.ia.sin_port) {
            if ( fromIn.ia.sin_addr.s_addr == this->from.ia.sin_addr.s_addr ) {
                return true;
            }
        }
    }
    return false;
}
`